backend/healthcheck.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
```

Comment:
Is there a way to exclude some files in linting process?

Reply:
I think the way you have it should disable those roles from the file

Reply:
Yup. I just think that it will be cleaner if we have some whitelist of files not to be processed by linter. But that's just me. If we're okay with the first lines of file disabling lints, then all good. :bow:

Reply:
Feel free to  create a `.eslintignore` file to the select files completely from the lint

Reply:
Created base ignore file